cmd/broadcaster-cli/app/keyset/topup/topup.go

```diff
@@ -30,19 +24,36 @@ var Cmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
+
 		logger := helper.GetLogger()
 
 		wocClient := woc_client.New(woc_client.WithAuth(wocApiKey), woc_client.WithLogger(logger))
-		fundingKeySet, _, err := helper.GetKeySetsKeyFile(keyFile)
+
+		keysFlag := viper.GetString("keys")
+		selectedKeys := strings.Split(keysFlag, ",")
+
+		keys, err := helper.GetPrivateKeys()
 		if err != nil {
-			return fmt.Errorf("failed to get key sets: %v", err)
+			return err
 		}
 
-		err = wocClient.TopUp(context.Background(), !isTestnet, fundingKeySet.Address(!isTestnet))
+		keySets, err := helper.GetKeySets(keys, selectedKeys)
 		if err != nil {
 			return err
 		}
```

Comment:
Just loud thinking: maybe it could be one function too? I mean we could have helper functions like:
`func GetPrivateKeys()`
`func GetKeySetsFor(keys map[string]string (...))`
`func GetKeySets()`

The first two would be for future flexibility, and the last one would be used in the currently existing implementation. What do you think?